### PR TITLE
Fixed namespaces used in "Add A Calculation Step" and improved wording

### PR DESCRIPTION
**Pull request by a-ast**
**Description**

* Teamwork assistant classes were moved to `Akeneo\Pim\WorkOrganization\TeamworkAssistant` namespace in Akeneo 4 but it wasn't reflected in the documentation
* Slashes must be doubled to be properly rendered in the markdown documentation
* Slight changes in wording: "allows to log" -> "allows logging"

**Definition Of Done**

| Q                                 | A
| --------------------------------- | ---
| Technical Review and 2 GTM        | Todo
| English Review and 1 GTM          | Todo

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

**Review by jmleroux — APPROVED** [review]

File: manipulate_pim_data/teamwork_assistant/calculation_step.rst [hunk 1]
```diff
@@ -2,7 +2,7 @@ Add A Calculation Step
 ======================
 
 To add a new step you just need to create a new Step Class that implements
-`PimEnterprise\Component\ActivityManager\Job\ProjectCalculation\CalculationStep\CalculationStepInterface` interface.
+`Akeneo\\Pim\\WorkOrganization\\TeamworkAssistant\\Componen\t\Job\\ProjectCalculation\\CalculationStep` interface.
```
Comment (tseho) [inline manipulate_pim_data/teamwork_assistant/calculation_step.rst:5]:
```suggestion
`Akeneo\\Pim\\WorkOrganization\\TeamworkAssistant\\Component\\Job\\ProjectCalculation\\CalculationStep` interface.
```

**Review by ValentinMumble — APPROVED** [review]

File: manipulate_pim_data/teamwork_assistant/calculation_step.rst [hunk 2]
```diff
@@ -2,7 +2,7 @@ Add A Calculation Step
 ======================
 
 To add a new step you just need to create a new Step Class that implements
-`PimEnterprise\Component\ActivityManager\Job\ProjectCalculation\CalculationStep\CalculationStepInterface` interface.
+`Akeneo\\Pim\\WorkOrganization\\TeamworkAssistant\\Component\\Job\\ProjectCalculation\\CalculationStep` interface.
```
Comment (mmetayer) [inline manipulate_pim_data/teamwork_assistant/calculation_step.rst:5]:
```suggestion
`Akeneo\\Pim\\WorkOrganization\\TeamworkAssistant\\Component\\Job\\ProjectCalculation\\CalculationStepInterface`.
```

Comment (tseho) [inline manipulate_pim_data/teamwork_assistant/calculation_step.rst:5]:
```suggestion
`Akeneo\\Pim\\WorkOrganization\\TeamworkAssistant\\Component\\Job\\ProjectCalculation\\CalculationStep\\CalculationStepInterface` interface.
```

Comment (tseho) [reply manipulate_pim_data/teamwork_assistant/calculation_step.rst:5]: It's `Akeneo\\Pim\\WorkOrganization\\TeamworkAssistant\\Component\\Job\\ProjectCalculation\\CalculationStep\\CalculationStepInterface`, there is a folder `CalculationStep`.

**Review by pierallard — APPROVED** [review]

File: manipulate_pim_data/teamwork_assistant/calculation_step.rst (hunk 2 above)
Comment (mmetayer) [reply manipulate_pim_data/teamwork_assistant/calculation_step.rst:5]: @tseho we should have synchronized :joy:

**Review by Doodoune — APPROVED** [review]

**Review by tseho — APPROVED** [review]

**Pull request merged**